tests/ctst/common/utils.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * This helper will dynamically extract a property from a CLI result
+ * @param {object} results - results from the command line
+ * @param {string[]} propertyChain - the property chain to extract, like Policy, Arn
+ * @return {string} - the expected property, or null if an error occurred when parsing results.
+ */
+export function extractPropertyFromResults(results: { err: null; stdout: string } | any, ...propertyChain: string[]) : any | null {
```

Comment:
According to the jsdoc, we return a string (or null), "any" seem wrong here

```suggestion
export function extractPropertyFromResults(results: { err: null; stdout: string } | any, ...propertyChain: string[]) : string | null {
```